osu.Game/Graphics/UserInterface/HoverSampleDebounceComponent.cs

```diff
@@ -5,19 +5,22 @@
 
 using osu.Framework.Allocation;
 using osu.Framework.Bindables;
-using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics;
 using osu.Framework.Input.Events;
 using osu.Game.Configuration;
+using osuTK;
 
 namespace osu.Game.Graphics.UserInterface
 {
     /// <summary>
     /// Handles debouncing hover sounds at a global level to ensure the effects are not overwhelming.
     /// </summary>
-    public abstract partial class HoverSampleDebounceComponent : CompositeDrawable
+    public abstract partial class HoverSampleDebounceComponent : Component
```

Comment:
I'm not sure I agree with this. `Component` is usually reserved for background processing components that need to be attached to the drawable hierarchy (most often in order to access DI) - but this really _is_ a drawable the way I look at it? It handles hover. It's not anything like those other usages.

At some point if/when the `Component : Drawable` inheritance inversion happens, this will likely cease to make any sense.

Reply:
https://github.com/ppy/osu/blob/7bc8908ca9c026fed1d831eb6e58df7624a8d614/osu.Game/Input/IdleTracker.cs#L20

`IdleTracker` is also *transparent* and inherits `Component` but tracks all input events with `Handle()`.

If the change gets reverted, I would either want to rename the "Component" part of `HoverSampleDebounceComponent`, or rename `Component` as the term is not properly defined.

Reply:
I'm also not sure about this. But I think it's fine until the point `Component` changes to be something more than what it is, at which point we'll no doubt check each usage (or it will be breaking change).